### PR TITLE
Default values in structure

### PR DESCRIPTION
**Pull request by ihippik**
Swagger default value for golang stucture.

format: 
`doc:"default(my_defaul_value)"`

```
type MyStruct struct {
	Template string `json:"template" doc:"default(<h1>Hello, {{ name }}! </h1>)"`
	Name string `json:"name"`
	MyNumber int64 `json:"temp"  doc:"default(10)"`
	MyBool bool `json:"bl"  doc:"default(true)"`
	MyFloat float64 `json:"fl"  doc:"default(12.1234)"`
}
```

### CODE REVIEW

**Comment by ihippik** [comment]
perform with:
https://github.com/astaxie/beego/pull/2211

**Comment by astaxie** [comment]
I think it also need to update the documents

File: g_docs.go [hunk 1]
```diff
@@ -770,6 +770,30 @@ func parseObject(d *ast.Object, k string, m *swagger.Schema, realTypes *[]string
...
+				
+				defaultValue := stag.Get("doc")
+				if defaultValue != ""{
+					r, _ := regexp.Compile(`default\((.*)\)`)
+					if r.MatchString(defaultValue) {
+						res := r.FindStringSubmatch(defaultValue)
+						mp.Default = res[1]
+						switch realType{
+							case "int","int64", "int32", "int16", "int8":
+								mp.Default, _ = strconv.Atoi(res[1])
```
Comment (astaxie) [inline g_docs.go]: I think we need to handler the error here and follow. As if the real type is int while the default value is not int. Then we need to panic it and tell the user which is wrong.

Comment (ihippik) [inline g_docs.go]: @astaxie Yes you are right. So you can not leave. That's what you do or do I add?

**Pull request merged**